package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/helix-cli",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
why is the package.json modified?

Reply:
CLI repo changed it already; so it became merge conflict!

Reply:
then I don't understand the diff here. can you rebase everything on master?

Reply:
good point! I should be able to.

Reply:
You know of a way to rebase and keep history?

Reply:
wait nevermind, thanks Tobi :)